x-pack/solutions/security/plugins/security_solution/public/timelines/components/duration/index.tsx

```diff
@@ -18,7 +18,7 @@ export const EVENT_DURATION_FIELD_NAME = 'event.duration';
  */
 export const Duration = React.memo<{
   contextId: string;
-  eventId: string;
+  eventId?: string;
```

Comment:
Q: Was this an intentional change of type?

Reply:
Same as https://github.com/elastic/kibana/pull/207878#discussion_r1933576523